AngleSharp.Io.Tests/Network/HttpClientRequesterTests.cs

```diff
@@ -0,0 +1,265 @@
+﻿namespace AngleSharp.Io.Tests.Network
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AngleSharp.Io.Network;
+    using AngleSharp.Network;
+    using AngleSharp.Services.Default;
+    using FluentAssertions;
+    using NUnit.Framework;
+    using NetHttpMethod = System.Net.Http.HttpMethod;
+    using AngleSharpHttpMethod = AngleSharp.Network.HttpMethod;
+
+
+    [TestFixture]
+    public class HttpClientRequesterTests
+    {
+        [Test]
+        public async Task RequestWithContent()
+        {
+            // ARRANGE
+            var ts = new TestState();
+
+            // ACT
+            await ts.Target.RequestAsync(ts.Request, CancellationToken.None);
+
+            // ASSERT
+            ts.HttpRequestMessage.Version.Should().Be(new Version(1, 1));
```

Comment:
Actually I never used the fluent assertions before, but they look pretty cool and expressive. Very nice!